pkg/platform/aws/aws.go

```diff
@@ -75,29 +72,35 @@ type config struct {
 	EnableReporting          bool              `hcl:"enable_reporting,optional"`
 	CertsValidityPeriodHours int               `hcl:"certs_validity_period_hours,optional"`
 	WorkerPools              []workerPool      `hcl:"worker_pool,block"`
+	// Raw fields that will store the strings after unmarshalling of
+	// SSHPubKeys, ControllerCLCSnippets, Tags and unmarshalling of
+	// some fields in WorkerPools
+	TagsRaw                  string
+	SSHPubKeysRaw            string
+	ControllerCLCSnippetsRaw string
+	WorkerPoolsListRaw       []map[string]string
```

Comment:
Comments from the other PR still apply here too: https://github.com/kinvolk/lokomotive/pull/355#discussion_r416812854. IMHO, this can be way simpler as mentioned there.